data/json/ascii_art/ammo/20x66mm.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "type": "ascii_art",
+    "id": "20x66_abstract",
```

Comment:
abstracts are not actual items, and don't need to have an ascii art

Reply:
the abstract doesnt but items that copy from it will use the art it has, which is useful for things like ammo where most of the variants look the same ^-^